frontend/src/app/components/register-page/register-page.component.ts

```diff
@@ -23,19 +24,22 @@ export class RegisterPageComponent implements OnInit {
   faGithub = faGithub;
   faGoogle = faGoogle;
 
-  constructor() { 
+  constructor(
+    public validationConstraints = ValidationConstraints,
+  ) { 
```

Comment:
Why you use constructor for this if `ValidationConstraints` is constants?

Reply:
Habit:D